src/components/create-deck/create-deck.js

```diff
@@ -0,0 +1,121 @@
+import EditPanel from '../edit-panel/edit-panel';
+import { Component } from 'react';
+
+import './create-deck.css';
+
+const store = require('store');
+
+class CreateDeck extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            eng: '',
+            rus: '',
+            deck: [],
+            deckTitle: ''
+        };
+        this.maxId = 1;
+    }
+
+    onValueChange = (e) => {
+        this.setState({
+            [e.target.name]: e.target.value
+        });
+    }
+
+    onAddToDeck = (eng, rus) => {
+        const newCard = {
+            eng,
+            rus,
+            id: this.maxId++
+        };
+        this.setState(({deck}) => {
+            const newDeck = [...deck, newCard];
+            return {
+                deck: newDeck
+            }
+        });
+    }
+
+    onAddDeckToLocalStorage = (key, values) => {
+        store.set(key, values);
```

Comment:
it's a nice component but for the future, it's better to use native methods localStorage.setItem
no need to fix it now